.env

```diff
@@ -0,0 +1,8 @@
+MYSQL_USER=root
+MYSQL_PASSWORD=Root123
+MYSQL_DATABASE=booksdb
+MYSQL_LOCAL_PORT=3307
+MYSQL_DOCKER_PORT=3306
+SPRING_LOCAL_PORT=8080
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```

Comment:
add `.env` to `.gitignore`. It is not secure (and should be removed from the remote repository). Better add .env.template with variables without values.